aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Pages/Products.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@page
+@model TagHelpersBuiltIn.Pages.ProductsModel
+@{
+    ViewData["Title"] = "Products";
+}
+
+<h2>Products</h2>
+
+<!-- <snippet_HtmlHelper> -->
```

Comment:
Side question: I assume that these can't be Razor commented out?

```cshtml
@* <snippet_HtmlHelper> *@
@foreach (var product in Model.Products)
{
    @await Html.PartialAsync("_ProductPartial", product)
}
@* </snippet_HtmlHelper> *@
```

Reply:
DocFX doesn't complain about that syntax; however, the snippet doesn't display when using Razor comments. It should work, and this syntax is preferred (since the comment doesn't display in the output). I'll log an issue for this one too.

Reply:
This is being tracked by https://github.com/dotnet/docfx/issues/3013